### PR TITLE
docs: add must-read and function boundary

### PR DESCRIPTION
**Pull request by vagetablechicken**

### CODE REVIEW

File: docs/zh/quickstart/function_boundary.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+### Deploy
+
+通过`DEPLOY <deploy_name> <sql>`可以上线sql，这个操作不仅是上线sql，也会自动解析sql帮助创建索引（可以通过`DESC <table_name>`查看索引详情）。
+
+但是，`DEPLOY`自动创建的索引不支持导入已有的数据。举例说明：
+
+表t1当前有两个索引index0和index1，插入10条数据后，index0和index1中都保存有10条数据。再`DEPLOY`某个sql，导致t1新增一个索引index2，此时的index2内是没有数据的。如果使用该deploy sql计算，将会使用index2，而它没有数据，逻辑上等同于一张空表。
+
+解决方案可以选择其一：
```
Comment (dl239) [inline docs/zh/quickstart/function_boundary.md:52]: 如果有长窗口不支持有数据的deploy

File: docs/zh/quickstart/function_boundary.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+集群版LOAD DATA无论导入到在线或离线，都是离线job。源数据的格式规则，离线在线没有区别。
+
+推荐使用hdfs文件作为源数据，无论taskmanager是local/yarn模式，还是taskmanager在别的主机上运行，都可以导入。如果源数据为`file`本地文件，那么需要考虑taskmanager模式和运行主机，是否可以顺利进行导入。
+```{seealso}
+1. taskmanager是local模式，`file`的本地就是指taskmanager进程所在的主机。只有将源数据放在taskmanager进程的主机上才能顺利导入。请注意你的taskmanager是本机运行，还是分布在别处主机。
+1. taskmanager是yarn(client and cluster)模式时，由于不知道运行容器是哪台主机，不可使用`file`作为源数据地址。
+```
+
+并且parquet格式，
```
Comment (dl239) [inline docs/zh/quickstart/function_boundary.md]: 多余了？

Comment (vagetablechicken) [reply docs/zh/quickstart/function_boundary.md]: yep

File: docs/zh/quickstart/openmldb_quickstart.md [hunk 3]
```diff
@@ -265,9 +264,16 @@ cd taxi-trip
...
 
-如果希望预览数据，用户亦可以使用 `SELECT` 语句，但是离线模式下该命令亦为非阻塞命令，查询结果需要查看日志（默认在/work/openmldb/taskmanager/bin/logs/jog_x.log，如需更改，修改taskmanager.properties的`job.log.path`）。
+如果希望预览数据，用户可以使用 `SELECT * FROM demo_table1` 语句，推荐`SELECT`前将离线命令设置为同步模式，即
+```sql
+SET @@sync_job=true;
+-- 如果数据较多容易超时（默认1min），请调大job timeout: SET @@job_timeout=600000;
+SELECT * FROM demo_table1;
+```
+
+非阻塞命令（异步模式）时，查询结果需要查看日志（默认在`/work/openmldb/taskmanager/bin/logs/jog_x.log`，如需更改日志的保存地址，修改`/work/openmldb/conf/taskmanager.properties`的`job.log.path`）。
```
Comment (dl239) [inline docs/zh/quickstart/openmldb_quickstart.md]: jog_x.log -> job_x.log 
jog_x_error.log -> job_x_error.log

File: docs/zh/quickstart/function_boundary.md (hunk 1 above)
Comment (vagetablechicken) [reply docs/zh/quickstart/function_boundary.md:52]: added

**Review by dl239 — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**